sdk/keyvault/keyvault-secrets/test/public/list.spec.ts

```diff
@@ -153,7 +150,7 @@ describe("Secret client - list secrets in various ways", () => {
     results.sort(comp);
     versions.sort(comp);
 
-    expect(results).to.deep.equal(versions);
```

Comment:
It does feel a little bit like, by exporting `assert` at the top level, we're promoting it to a higher status than the BDD API and basically saying "we prefer assert over expect." It takes a little more doing to import `expect` and `should`. Personally I use `assert`, but maybe we have some BDD fans among us that might like to see `expect` and `should` at the top level as well. I don't know.

Reply:
Yeah I agree. I'm one of the people that prefers BDD / expect syntax but I would rather be consistent so I switched to `assert`. Do you think we should export both as a top-level export?

Reply:
I did end up exporting both assert and expect

Reply:
what about `should`?

Reply:
I can export that too if folks are using it - I honestly have only see assert and expect used in our packages. Do you think I should add `should`?

Reply:
It is used in messaging libs (amqp, service bus, and event hubs).  Yes it would be great to export it.

Reply:
@jeremymeng added!